src/persist/src/metrics.rs

```diff
@@ -73,6 +73,7 @@ pub struct PostgresConsensusMetrics {
     pub(crate) connpool_size: UIntGauge,
     pub(crate) connpool_acquires: IntCounter,
     pub(crate) connpool_acquire_seconds: Counter,
+    pub(crate) connpool_available: prometheus::Gauge,
```

Comment:
can we use UIntGauge instead so we don't have to cast u64->f64?

Reply:
Unfortunately it's `i64` :/. A bit weird, but it seemed safest to pipe it through in close to its original form.

Reply:
Wild! Yeah, what you have lgtm then